docs/sources/mimir/references/http-api/index.md

```diff
@@ -566,8 +566,17 @@ GET,POST <prometheus-http-prefix>/api/v1/cardinality/label_values
 Returns realtime label values cardinality associated to request param `label_names[]` across all ingesters, for the authenticated tenant, in `JSON` format.
 It returns the series count per label value associated to request param `label_names[]`.
 
-As far as this endpoint generates cardinality report using only values from currently opened TSDBs in ingesters, two subsequent calls may return completely different results, if ingester did a block
-cutting between the calls.
+Two methods of counting are available, `inmemory` and `active`. This choice is controlled via the `count_method` parameter.
+
+The `inmemory` counting method counts the number of series using all values from currently opened TSDBs in ingesters.
+Two subsequent calls may return completely different results, if ingester did a block cutting between the calls.
+This method of counting is most useful for understanding ingester memory usage.
+
+The `active` counting method counts also uses values from currently opened TSDBs in ingesters, but it filters out series that have not received a sample within active series idle timeout.
```

Comment:
```suggestion
The `active` method also counts series in currently opened TSDBs in Mimir's ingesters, but filters out series that have not received a sample within a configurable duration of time. 
```

Reply:
honestly I still don't understand how if `active` is just a subset of `inmemory` (you said it "filters stuff out" iiuc) how its not also impacted by this same block cutting problem? But I don't necessarily think I need to understand this =)

Reply:
Thank you for explaining!